CHANGELOG.md

```diff
@@ -1,9 +1,5 @@
 # Changelog
 
-## 0.13.0
-
-- **Add:** add `driving-traffic` profile to Isochrone service.
-
```

Comment:
We need to keep this changelog entry because 0.13.0 has already been released. We will also want to add to the changelog right below line 2:

```md
## Unreleased

- **Revert:** add `driving-traffic` profile to Isochrone service.
```

Reply:
Thank you for this correction!

Updated in https://github.com/mapbox/mapbox-sdk-js/pull/421/commits/e813eb87a6fae2fa9d6b182d866c1727a7abe713#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL3-R10